eks-without-vpc/variables.tf

```diff
@@ -0,0 +1,82 @@
+#
+# Variables Configuration
+#
+
+variable "cluster-name" {
+  type    = "string"
```

Comment:
Also a default value for cluster name of `spinnaker` can be good here

Reply:
done